fluid-slider/src/main/kotlin/com/ramotion/fluidslider/FluidSlider.kt

```diff
@@ -369,7 +388,22 @@ class FluidSlider @JvmOverloads constructor(context: Context, attrs: AttributeSe
         canvas.drawOval(rectLabel, paintLabel)
 
         val text = bubbleText ?: (position * 100).toInt().toString()
-        drawText(canvas, paintText, text, Paint.Align.CENTER, colorBubbleText, 0f, rectLabel, rectText)
+        if (text.length > 3)
```

Comment:
Why only text.length >3 ?

Reply:
because only when the number have more than 3 digits it will start overflow the ball space.